test/e2e/framework/framework.go

```diff
@@ -95,6 +96,17 @@ type Framework struct {
 	// Federation specific params. These are set only if federated = true.
 	FederationClientset_1_5 *federation_release_1_5.Clientset
 	FederationNamespace     *v1.Namespace
+
+	// CRI client
+	CRIClient *InternalApiClient
```

Comment:
@Random-Liu True, CRI validation test is not associated with existing e2e test framework. So should we add another independent framework(which is specially for CRI validation test) like e2e framework? And should we put these files in `/test` or in `/kubelet`?